docs/apache-airflow/templates-ref.rst

```diff
@@ -75,6 +75,8 @@ Variable                                    Description
 ``{{ dag_run }}``                           A reference to the DagRun object.
 ``{{ test_mode }}``                         Whether the task instance was called using the CLI's test
                                             subcommand.
+``{{ expanded_ti_count }}``                 Number of task instances that a mapped task was expanded into. If
```

Comment:
I guess we should we add "new in 2.5" somewhere?

Reply:
I thought about that but it doesn’t seem to fit very well with the current format, and many other variables here (e.g. the data interval ones) don’t annotate when they were added either. Maybe I can revamp the entire table?

Reply:
I restructured the entire table.